scipy/linalg/tests/test_decomp_polar.py

```diff
@@ -2,7 +2,7 @@
 from numpy.linalg import norm
 from numpy.testing import (assert_, assert_allclose, assert_equal)
 from scipy.linalg import polar, eigh
-
+import pytest
```

Comment:
minor: convention is to have imports from pytest/the standard library/other utilities at the top, then imports from `numpy`, then imports from `scipy`. Maybe move this to line 1.